src/Dfc.CourseDirectory.Core/Models/ApprenticeshipLocationType.cs

```diff
@@ -15,9 +15,9 @@ public static class ApprenticeshipLocationTypeExtensions
         public static string ToDescription(this ApprenticeshipLocationType apprenticeshipLocationType) =>
             apprenticeshipLocationType switch
             {
-                ApprenticeshipLocationType.EmployerBased => "Employer based locations",
-                ApprenticeshipLocationType.ClassroomBased => "Your locations",
-                ApprenticeshipLocationType.ClassroomBasedAndEmployerBased => "Your location and employer locations",
+                ApprenticeshipLocationType.EmployerBased => "Employer based venues",
```

Comment:
I don't think these three should be changed. AFAIK we're sticking with 'location' for apprenticeships